assets/js/modules/analytics/common/account-create.js

```diff
@@ -162,10 +162,8 @@ export default function AccountCreate() {
 				<TimezoneSelect />
 			</div>
 
-			<p>
-				{ hasProvisioningScope && __( 'You will be redirected to Google Analytics to accept the terms of service.', 'google-site-kit' ) }
-				{ ! hasProvisioningScope && __( 'You will need to give Site Kit permission to create an Analytics account on your behalf and also accept the Google Analytics terms of service.', 'google-site-kit' ) }
-			</p>
+			{ hasProvisioningScope && <p>{ __( 'You will be redirected to Google Analytics to accept the terms of service.', 'google-site-kit' ) }</p> }
+			{ ! hasProvisioningScope && <p>{ __( 'You will need to give Site Kit permission to create an Analytics account on your behalf and also accept the Google Analytics terms of service.', 'google-site-kit' ) }</p> }
```

Comment:
Now that there's no change needed here, would you please restore it to the same in `develop` ?

Reply:
@aaemnnosttv there is a change: now text is wrapped with `<p>` tags which help us to select it in the `toMatchElement`.

Reply:
It was also wrapped in a `<p>` tag before, it's only the content of the tag which changes based on `hasProvisioningScope`.

Reply:
Ah... yes, you are right... Let me revert it.

Reply:
Should be good now.